src/react/components/searchPage.js

```diff
@@ -0,0 +1,398 @@
+          query,
+          params: {
+            facets: ["siteName", "fileType"],
+            facetFilters: siteNameFilters,
+          },
+        },
+        // Disjunctive query for fileType facet values.
+        {
+          indexName: "crawler_federated-search",
+          query,
+          params: {
+            facets: ["siteName", "fileType"],
+            facetFilters: fileTypeFilters,
+          },
+        },
+      ])
+      .then((queryResults) => {
+        setResults(queryResults.results[0]);
+        setSiteNameValues(queryResults.results[1].facets.siteName);
+        setFileTypeValues(queryResults.results[2].facets.fileType);
+      });
+  };
+
+  // Listen for changes to query, pager, or facets and update search results.
+  useEffect(() => {
+    updateSearchResults();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [query, page, selectedFacets]);
+
+
+  const wrapperClasses = `flex-grow w-auto border-0 border-b border-solid`;
+
+  const clearBtnClasses = `flex items-center bg-transparent focus:bg-transparent hover:bg-transparent focus:text-black hover:underline focus:underline text-m0 font-semibold border-none p-0 mr-8 md:mr-10 mt-2`;
+
+  const inputClasses = `border-0 w-full flex-1 rs-px-1 py-10 outline-none focus:ring-0 focus:ring-transparent focus:shadow-none`;
+
+  const submitBtnClasses = `flex items-center justify-center w-16 h-16 rounded-full transition-colors ml-4`;
+
+  const autocompleteLinkClasses = `cursor-pointer font-regular inline-block w-full no-underline px-6 py-4 rounded-full`;
+
+  const autocompleteLinkFocusClasses = ``;
+
+  const autocompleteContainerClasses = `absolute p-4 shadow-md w-full border`;
+
+  const facets = results.facets && (
+    <React.Fragment>
+      {siteNameValues && (
+        <SearchFacet
+          label="Sites"
+          attribute="siteName"
+          facetValues={siteNameValues}
+          selectedOptions={selectedFacets.siteName}
+          onChange={(values) => updateSiteFacet(values)}
+          className={!!selectedFacets.siteName.length && "mb-8"}
+          exclude={["YouTube", "SoundCloud", "Apple Podcasts"]}
+        />
+      )}
+      {fileTypeValues && (
+        <SearchFacet
+          label="Media"
+          attribute="fileType"
+          facetValues={fileTypeValues}
+          selectedOptions={selectedFacets.fileType}
+          onChange={(values) => updateFileTypeFacet(values)}
+          optionClasses="capitalize"
+          className="mb-8"
+          exclude={["html", "pdf"]}
+        />
+      )}
+    </React.Fragment>
+  );
+
+  return (
+    <div>
+      <section
+          className="aloglia-search--title-banner w-full py-32 md:py-70 xl:py-108 text-center flex-wrap"
+        >
+        <h1 className="font-bold mb-0 mt-0">
+          Search for...
+        </h1>
+      </section>
+      <section className="algolia-search--main flex-12-of-12 w-full">
+        <div className="grid grid-cols-12 relative bassefont-19">
```

Comment:
```suggestion
        <div className="grid grid-cols-12 relative lg:gap-x-20 centered-container">
```
One more. The search input field on the search page needs the centered-container as well, and the grip gap so it lines up with the section below. That'll be all! 🎉

Reply:
@sherakama I'm going to commit this change in